wizard.py

```
@@ -6,728 +6,581 @@
-                with open(config_yml_path, 'r') as f:
+                with open(config_yml_path, "r") as f:
                     config_data = yaml.safe_load(f)
-                    obsidian_config = config_data.get('memory', {}).get('obsidian', {})
-                    obsidian_enabled = obsidian_config.get('enabled', False)
+                    obsidian_config = config_data.get("memory", {}).get("obsidian", {})
+                    config_obsidian_enabled = obsidian_config.get("enabled", False)
             except Exception as e:
                 console.print(f"[yellow]Warning: Could not read config.yml: {e}[/yellow]")
 
-    # Final Summary
     console.print(f"\n🎊 [bold green]Setup Complete![/bold green]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)